### PR TITLE
Add operator<< support to UDAValue

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/parser/eclipse/Deck/UDAValue.hpp [hunk 1]
```diff
@@ -23,6 +23,7 @@
 #include <stdexcept>
 #include <vector>
 #include <string>
+#include <iostream>
```
Comment (bska) [inline opm/parser/eclipse/Deck/UDAValue.hpp]:
```C++
#include <iosfwd>
```
You don't need `<iostream>` here.

Comment (joakim-hove) [reply opm/parser/eclipse/Deck/UDAValue.hpp]: Fixed - never heard of `isofwd` before.

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/parser/eclipse/Deck/UDAValue.hpp (hunk 1 above)
Comment (bska) [reply opm/parser/eclipse/Deck/UDAValue.hpp]: > Fixed

Thanks!

> never heard of `isofwd` before.

The header, sadly, is unknown to many (myself included until fairly recently).  It was created for this exact purpose&mdash;to forward declare IO-stream types used in API declarations so client code wouldn't have to parse the big `<iostream>` header each time they included the header declaring the APIs.  Of course, **using** the APIs in an implementation file will then often require also including `<iostream>` (or the more focused headers `<istream>`, `<ostream>` or `<fstream>`), but it's less of a concern in `.cpp` files.

**Pull request merged**